cmp/ag-grid/AgGridModel.js

```diff
@@ -324,7 +334,36 @@ export class AgGridModel {
         // Remove invalid columns. ag-Grid does not like calls to set state with unknown column IDs.
         columns = columns.filter(it => validColIds.includes(it.colId));
 
-        agColumnApi.setColumnState(columns);
+        agColumnApi.applyColumnState({state: columns, applyOrder: true});
+    }
+
+    /**
+     * Sets the sort state on the grid's column state
+     * @param {GridSorter[]} sortBy
+     */
+    applySortBy(sortBy) {
+        const {agColumnApi} = this,
+            cols = agColumnApi.getColumnState(),
+            sortedCols = sortBy.
+                filter(sorter => cols.find(it => it.colId === sorter.colId)).
```

Comment:
lodash find works great in situations like this

find (cols, {colId: sorter.colId})

Reply:
Done.